src/libs/ReportUtils.js

```diff
@@ -1900,6 +1904,8 @@ function updateReportPreview(iouReport, reportPreviewAction) {
                 type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
             },
         ],
+        childLastMoneyRequestComment: comment || reportPreviewAction.childLastMoneyRequestComment,
+        childMoneyRequestCount: reportPreviewAction.childMoneyRequestCount + 1,
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/26767
We shouldn't increase `childMoneyRequestCount` if you selected "Pay Elsewhere"
More details [here](https://github.com/Expensify/App/issues/26767#issuecomment-1706737988)